gramps/gen/datehandler/_datestrings.py

```diff
@@ -335,13 +335,13 @@ def __init__(self, locale):
     except AttributeError:
         localized_months = old_long
 
-    def print_po_snippet(en_loc_old_lists, context):
+    def print_po_snippet(en_loc_old_lists, context, line: int = 10000):
```

Comment:
Yes, it does work the same way. I am pretty sure my slight refactor (needed for mypy checks to pass) does not affect the way it was implemented before. That being said, the function as such is quite a terrible way to do things, but my goal in this PR is to do minimal changes without affecting functionality.

Reply:
No, it does not; `print_po_snippet.line` is effectively global state, so at the end of the function, it will be increased by however number of times the `+= 1` is called, and remain that way for the next call. In the new code, the default value for parameter is _not_ saved anywhere, and begins at 10000 for every call.